### ISSUE TITLE
Series overlaps axis when more than 3 series are boosted

### ISSUE THREAD

**Issue by BuckyMaler** [highcharts/highcharts#7250]
#### Expected behaviour
No overlapping between axis and series when chart is rendered, regardless of the number of boosted series.

#### Actual behaviour
When more than 3 series are boosted (version 5.0.14) the chart renders with overlapping series and axis.

#### Live demo with steps to reproduce
Due to time constraints I'm unable to produce a demo at this time. I hope to update the issue soon. I cannot point to a repo because this is company software and not available on GitHub.

We are currently using Highcharts v5.0.14 with `boost-canvas` and `boost` modules imported/required. The specified `boostThreshold` is the default, 5000. When rendering more than 3 series that are boosted our chart results in series overlapping axis.

#### Affected browser(s)
Latest Chrome, Firefox, Safari, and IE 11

![series-overlaps-axis chrome](https://user-images.githubusercontent.com/14145352/31404801-b17ee54a-adc2-11e7-95ea-f228f2608f20.png)

**Comment by pawelfus** [comment]
Hi @BuckyMaler 

Thank you for reporting, could you try also v6 ? I tried to recreate issue with v5, but it looks fine: http://jsfiddle.net/yqaLx4k9/1/ (well, tooltip is not working, but it's fixed in v6 already).

Live demo (like jsFiddle) would be great.

**Comment by BuckyMaler** [comment]
Hi @pawelfus,

Thanks for posting the demo. I tried v6 and encountered the same error.

We had a release this past week, so my team and I have been quite busy. Apologies for the delayed response.

The code for this chart has become rather complex and unfortunately fragile, and by no means is that due to Highcharts. We made the decision to disable boost on this chart for the time being with hopes that we'll simplify our logic in the near future and reapply boost to a more readable and extensible component. Feel free to close this out as I don't know when I'll be returning to this.
